src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldTransformers/HtmlFieldTransformer.js

```diff
@@ -10,7 +10,7 @@ export default class HtmlFieldTransformer implements FieldTransformer {
             return null;
         }
 
-        const sanitizedHtml = sanitizeHtml(value, {
+        const sanitizedHtml = sanitizeHtml(value.toString(), {
```

Comment:
The new release also changed the behaviour for numbers and we need todo a `toString()` here now: https://github.com/apostrophecms/sanitize-html/issues/593